homeassistant/helpers/condition.py

```diff
@@ -606,23 +606,37 @@ def sun(
 
     if sunrise is None and SUN_EVENT_SUNRISE in (before, after):
         # There is no sunrise today
+        condition_trace_set_result(False, message="no sunrise today")
```

Comment:
We should set `sun.sun` as the linked entity ID, that way we can teach the frontend that it can just look at the `entity_id` field in the condition trace result to find the entities it should show.

Reply:
The attributes in `sun.sun` are not enough, unfortunately.